FastpackDevserver/Main.re

```diff
@@ -0,0 +1,52 @@
+module C = Cohttp_lwt_unix;
+module CServer = Cohttp_lwt_unix.Server;
+
+let serveStatic = (base, path) => {
```

Comment:
This is mostly borrowed from https://github.com/anmonteiro/reason-graphql-fullstack